modules/Media.js

```diff
@@ -16,7 +16,8 @@ class Media extends React.Component {
     ]).isRequired,
     render: PropTypes.func,
     children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
-    targetWindow: PropTypes.object
+    targetWindow: PropTypes.object,
+    onQueryStateChange: PropTypes.func
```

Comment:
Don't forget to rename it to `onChange`, as per @mjackson's [comment](https://github.com/ReactTraining/react-media/issues/94#issuecomment-415994237).